### DISCUSSION TITLE
夜间模式选项

### DISCUSSION THREAD

**Discussion by dwdchen** [the1812/Bilibili-Evolved#4902] [功能建议 / Ideas]
希望能增加夜间模式仅对功能和设置页生效的选项

**Comment by the1812** [comment]
为啥需要这种效果?

**Reply by Geeyun-JY3** [reply]
大胆猜测一下，可能是不想用本脚本的`夜间模式`，但仍希望本脚本的功能和设置面板能有深色模式（夜间模式），同时通过其他方式让破站界面有了深色模式（夜间模式）。

**Comment by GVD20** [comment]
[Bilibili-Evolved](https://github.com/the1812/Bilibili-Evolved)在启用深色模式组件后会自动覆盖页面的样式，这本来没有什么问题，但当我希望使用另一个插件（如 [BewlyBewly](https://github.com/BewlyBewly/BewlyBewly) 或 Stylus)控制样式时，Bilibili-Evolved对页面样式表的控制仍然生效，这可能会导致冲突或者样式不统一的问题，如果关闭Bilibili-Evolved深色模式，设置页面和弹窗又会变成白色，仍然有样式不统一的问题
